system_tasks.yml

```diff
@@ -15,71 +19,134 @@
+- name: Add jessie-backports to apt configuration
+  apt_repository:
+    repo: deb http://ftp.debian.org/debian jessie-backports main
+    state: present
+    update_cache: yes
+
+- name: Install EFF Certbot
+  apt:
+    name: certbot
+    state: present
+    default_release: jessie-backports
+
+- name: Generate Letsencrypt certs
+  shell: "certbot certonly --standalone -n -d piecewise.opentechinstitute.org -m critzo@opentechinstitute.org --agree-tos"
+
+- name: Enable nginx default site
+  file:
+    src: "/etc/nginx/sites-available/default"
+    dest: "/etc/nginx/sites-enabled/default"
+    state: link
+
 - name: Fetch telescope
-  git: repo=https://github.com/m-lab/telescope.git
-       dest=/opt/telescope/
+  git:
+    repo: https://github.com/m-lab/telescope.git
+    dest: /opt/telescope/
+
 - name: Get latest MaxMind CSV
-  get_url: url=http://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum2.zip
-           dest=/tmp/GeoIPASNum2.zip
+  get_url:
+    url: http://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum2.zip
+    dest: /tmp/GeoIPASNum2.zip
+
 - name: Unzip MaxMind CSV
-  unarchive: src=/tmp/GeoIPASNum2.zip
-             dest=/tmp/
-             creates=/tmp/GeoIPASNum2.csv
-             copy=no
+  unarchive:
+    src: /tmp/GeoIPASNum2.zip
+    dest: /tmp/
+    creates: /tmp/GeoIPASNum2.csv
+    copy: no
 
 # The command below tells Ansible where to download or copy your customized Piecewise files
 # when it deploys your VM. Change the **repo** to match your fork and **version** to match 
 # your branch.
 #
+# Note that the example below is fictious. Use the github URL of your fork and your branch. 
+#
 #    For example, change:
-#       git: repo=https://github.com/opentechinstitute/piecewise.git
-#       dest=/opt/piecewise.git/
-#       version=master
+#       git: 
+#          repo: https://github.com/opentechinstitute/piecewise.git
+#          dest: /opt/piecewise.git/
+#          version: master
 #    to:
-#       git: repo=https://github.com/baltimore-github-org/piecewise.git
-#       dest=/opt/piecewise.git/
-#       version=master
+#       git: 
+#         repo: https://github.com/baltimore-github-org/piecewise.git
+#         dest: /opt/piecewise.git/
+#         version: master
 #
-# Note that the example above is fictious. Use the github URL of your fork and your branch. 
 - name: Fetch piecewise
-  git: repo=https://github.com/opentechinstitute/piecewise.git 
-       dest=/opt/piecewise.git/
-       version=master
+  git:
+    repo: https://github.com/opentechinstitute/piecewise.git
+    dest: /opt/piecewise.git/
+    version: master
 
 - name: Deploy piecewise
-  file: src=/opt/piecewise.git/{{ item.src }} dest=/opt/{{ item.dest }} state=link
+  file:
+    src: /opt/piecewise.git/{{ item.src }}
+    dest: /opt/{{ item.dest }}
+    state: link
   with_items:
       - { src: piecewise, dest: piecewise }
```

Comment:
Since `src` and `dest` are the same, I think this can be simplified by just having them like this:

```
with _items:
  - piecewise
  [...]
```